docs/user-guide/creating-application/secrets.md

```diff
@@ -37,7 +37,16 @@ There are five Data types that you can use to save your secret.
 
 Specify the Volume Mount folder path in `Volume Mount Path`, a path where the data volume needs to be mounted. This volume will be accessible to the Containers running in a pod.
 
-![](../../.gitbook/assets/secret3.jpg)
+![](../../.gitbook/assets/secret-mountpath-1.png)
+
+## Sub Path
+For multiple files mount at the same location you need to check sub path `bool` field, it will use the file name (key) as sub path. 
```

Comment:
To mount mutiple files from different configmap and secret at the same location you need to...